packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -236,6 +264,10 @@ class MessageGroup extends Widget<Properties> {
     }
   }
 
+  _isAlignmentStart(alignment: string): boolean {
+    return alignment === 'start';
```

Comment:
Please also move const { alignment } = this.option(); line to this method: you will reduce the number of copy paste

Reply:
good point